socs/agents/generator/agent.py

```diff
@@ -1,5 +1,6 @@
 import argparse
 import os
+import os.path
```

Comment:
I'm pretty sure we don't need this if `os` is already imported

Reply:
Agreed, removed.